docs/source/reference/faq.rst

```diff
@@ -93,6 +93,41 @@ To launch a VS Code tunnel using a SkyPilot task definition, you can use the fol
 
 Note that you'll be prompted to authenticate with your GitHub account to launch a VS Code tunnel.
 
+How to launch VMs in a subset of regions only (e.g., Europe only)?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When defining a task, you can use the ``resources.any_of`` field to specify a set of regions you want to launch VMs in.
+
+For example, to launch VMs in Europe only (which can help with GDPR compliance), you can use the following task definition:
+
+.. code-block:: yaml
+
+  resources:
+    # SkyPilot will perform cost optimization among the specified regions.
+    any_of:
+      # AWS:
+      - region: eu-central-1
+      - region: eu-west-1
+      - region: eu-west-2
+      - region: eu-west-3
+      - region: eu-north-1
+      # GCP:
+      - region: europe-central2
+      - region: europe-north1
+      - region: europe-southwest1
+      - region: europe-west1
+      - region: europe-west10
+      - region: europe-west12
+      - region: europe-west2
+      - region: europe-west3
+      - region: europe-west4
+      - region: europe-west6
+      - region: europe-west8
+      - region: europe-west9
+      # Or put in other clouds' Europe regions.
```

Comment:
In the future, we may want to support some predefined areas, such as
```
resources:
  region: gdpr-area
```

Also, this task yaml will make it quite complicated to also select some accelerators, as we have to add the same accelerator for all the items in `any_of`. In the future, we should be able to take default fields and additional `any_of`, i.e.:
```yaml
resources:
  accelerators: V100
  any_of:
      # AWS:
      - region: eu-central-1
      - region: eu-west-1
      - region: eu-west-2
      - region: eu-west-3
      - region: eu-north-1
      ...
```

Reply:
Yep, taking defaults + treating `any_of` and `ordered` as overrides sounds good.